entity-framework/core/providers/sqlite/limitations.md

```diff
@@ -17,6 +17,25 @@ The common relational library (shared by Entity Framework relational database pr
 * Sequences
 * Computed columns
 
+## Query limitations
+
+SQLite does not have native support for the following data types. You can read and write values for them using EF Core, and query query for equality (`where e.Property == value`); however, ordering, comparison, and other operations on them will need to be performed on the client.
```

Comment:
query query

Reply:
comparison other than equality?
does not equal work?

Reply:
Not equals works, but that's an equality operaiton.

Comparisons don't work. They may work within certain ranges of values, but not across the full range.